src/pages/StyleGuide.js

```diff
@@ -1,38 +1,52 @@
 import { div, h1 } from 'react-hyperscript-helpers'
+import { buttonPrimary, link, search, textInput } from 'src/components/common'
+import { validatedInput } from 'src/components/input'
 import * as Nav from 'src/libs/nav'
-import { buttonPrimary, textInput, link, search } from 'src/components/common'
+import { Component } from 'src/libs/wrapped-components'
```

Comment:
TOL: I don't like this wrapped `Component`. As a general rule, we should be aiming to _reduce_ the amount and frequency of state, whereas this pattern seems to encourage more of it. At the very least, I think all state should be explicitly initialized in a constructor, so it's obvious to readers.